source/overview/faq.rst

```diff
@@ -265,29 +269,157 @@ Working for Mattermost
+
+A source available license gives access to source code, but places restrictions on its use. The Mattermost Source Available License allows free-of-charge and unrestricted use of the source code in development and testing environments, but requires a valid Mattermost Enterprise E20 License in a production environment.
+
+How can I identify code licensed as source available?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the Mattermost Source Available LICENSE file appears at the root of a repository, the restrictions apply to all source code within the repository. A note in the README.md often identifies the use of this license and links to this FAQ.
+
+When the Mattermost Source Available LICENSE file appears in a specific directory, the restrictions apply to all source code within that directory. This directory is often called enterprise. For additional clarity, an additional LICENSE.enterprise file may appear in the root directory, symlinked to the enterprise/LICENSE file.
+
+In all cases, any third party components remain licensed under their original license.
+
+An example directory layout is shown below:
+
+.. image:: ../images/source-available-license.png
+
+Why are you changing the licensing model?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our plugin framework delivers substantial value to our enterprise customers, but requires significant development and support resources. This change to the licensing model allows us to continue developing open source features while selectively charging for features.
+
+How are repositories changing?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As we add enterprise-only functionality, we will update the license on affected Mattermost authored plugin repositories. The intent is to work alongside the existing, open source functionality in our plugins while reserving certain enterprise functionality to customers who pay us for enterprise licenses.
+
+To which repositories does this apply?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We plan to apply this license to the enterprise directories of our Jira, Microsoft O365 Calendar, and MS Teams Meetings plugins. We also intend to release our Incident Response and Channel Export plugins, currently in beta, under the Mattermost Source Available License. New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the README file of the GitHub repository and as part of any release notes. 
+
+We expect to keep plugins without an enterprise component under our open source license. No licensing changes are planned to non-plugin repositories, such as mattermost-server or mattermost-webapp.
+
+Will the repositories be public?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, existing repositories will stay public. We are now also able to make public several enterprise-only plugins previously developed in private.
```

Comment:
```suggestion
Yes, existing repositories will stay public. We are now also able to make public several enterprise-only plugins under the Mattermost Source Available License previously developed in private.
```

Reply:
Just to clarify they are being made public under the source available license, not under open source license.